BrownianMotion/StochasticIntegral/ApproxSeq.lean

```diff
@@ -21,6 +21,18 @@ variable {ι Ω E : Type*} [TopologicalSpace ι] [TopologicalSpace E]
 abbrev _root_.Function.RightContinuous [PartialOrder ι] (X : ι → Ω → E) :=
   ∀ ω a, ContinuousWithinAt (X · ω) (Set.Ioi a) a
 
+/-- A stochastic process has left limits if each of its realizations has left limits
+at all point. -/
+abbrev _root_.Function.HasLeftLimits [Preorder ι] (X : ι → Ω → E) :=
+  ∀ ω a, ∃ x, Filter.Tendsto (X · ω) (𝓝[<] a) (𝓝 x)
+
+/-- A càdlàg stochastic process maps compact sets to bounded sets. -/
+lemma _root_.isBounded_image_of_cadlag_of_isCompact {E : Type*} [LinearOrder ι]
+    [FirstCountableTopology ι] [PseudoMetricSpace E] {X : ι → Ω → E} (hX1 : RightContinuous X)
+    (hX2 : HasLeftLimits X) (ω : Ω) {s : Set ι} (hs : IsCompact s) :
+    IsBounded ((X · ω) '' s) := by
+  sorry
```

Comment:
I added a cadlag definition in the StochasticIntegral/Cadlag file. Could you use it? Perhaps you want a definition `IsCadlagProcess X := ∀ ω, cadlag (X · ω)`. Or refactor/delete my cadlag def if you prefer, as you think is best.